README.md

```diff
@@ -3,3 +3,21 @@
 Earlier this year Shopify announced Hydrogen, a web development framework optimized for building custom storefronts by Shopify. In this workshop you'll get a first look at Hydrogen and together we'll build a custom storefront using React Server Components, Vite and Tailwind CSS.
 
 Get started by [forking this repository on Stackblitz](https://stackblitz.com/github/Shopify/hydrogen-workshop).
+
+## Branches
+
+We'll be adding several features to our Hydrogen app throughout this workshop.
+Each feature corresponds to a new branch.
+If you get lost at any time, or want to start from scratch, you can fork off of these branches.
+
+| Feature | Branch | Stackblitz |
+| ------- | ------ | ---------- |
+| Starting | `main` | [Stackblitz](https://stackblitz.com/github/Shopify/hydrogen-workshop) |
+| Head component | `head-component` | [Stackblitz](https://stackblitz.com/github/Shopify/hydrogen-workshop/tree/head-component) |
+| Product list | `list-products` | [Stackblitz](https://stackblitz.com/github/Shopify/hydrogen-workshop/tree/list-products) |
+| Load more products | `load-more` | [Stackblitz](https://stackblitz.com/github/Shopify/hydrogen-workshop/tree/load-more) |
+| Product page | `product-page` | [Stackblitz](https://stackblitz.com/github/Shopify/hydrogen-workshop/tree/product-page) |
+| Cart | `cart` | [Stackblitz](https://stackblitz.com/github/Shopify/hydrogen-workshop/tree/cart) |
+| Custom responses | `custom-responses` | [Stackblitz](https://stackblitz.com/github/Shopify/hydrogen-workshop/tree/custom-responses) |
+
+> Disclaimer: If you don't want to use Stackblitz, or are having difficulties using it, you can clone the [repository](https://github.com/shopify/hydrogen-workshop). Then run `yarn` and `yarn dev` to start it up, or `npm install` and `npm run dev`.
```

Comment:
I've included this on the off chance that Stackblitz is being finicky for some participants.